dask_cuda/proxy_object.py

```diff
@@ -436,6 +436,23 @@ def __setattr__(self, name, val):
             else:
                 object.__setattr__(self._obj_pxy_deserialize(), name, val)
 
+    def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
+        inputs = tuple(
+            o._obj_pxy_deserialize() if isinstance(o, ProxyObject) else o
+            for o in inputs
+        )
+        kwargs = dict(
```

Comment:
Just a nitpick, I don't think you need the `dict` here.

Reply:
removed it 👍